src/content/docs/features/state.md

```diff
@@ -28,17 +28,36 @@ declare module "membrane" {
 import { state } from "membrane";
 //       ^^^^^
 
-export interface State {
-  count: number;
-}
-
 state.count ??= 0;
 
 export function count() {
   state.count++;
 }
 ```
 
+You can also provide type information for `state` by exporting a `State` type or interface.
```

Comment:
Since this pattern is a bit of a hack (i.e. exporting a type so that it gets returned to use from the membrane module) I think we should probably call that out here. It can be left as a todo

Reply:
I may be missing something. What is the point of calling out the implementation? Does it affect the DX at all? If not I feel like it's just noise to know that `membrane.d.ts` is importing from `yourProgram/index.ts`.